services/consensus_update_topic.proto

```diff
@@ -87,4 +88,19 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+      * Access control for update/delete of custom fees. Null if there is no key.
+      */
+    Key fee_schedule_key = 10;
+
+    /**
+      * If the transaction contains a signer from this list, no custom fees are applied.
+      */
+    repeated Key free_messages_key_list = 11;
+
+    /*
+     * The custom fee to be assessed during a message submission to this topic
+     */
```

Comment:
Text should specify behavior of the field and follow correct formatting:
```suggestion
    /**
     * A set of custom fee definitions.<br/>
     * These are fees to be assessed for each submit to this topic.
     * <p>
     * If this list is empty, the current set of custom fees
     * SHALL NOT change.<br/>
     * If this list is not empty, the current `fee_schedule_key` MUST sign
     * this transaction.<br/>
     * If this list is not empty, each fee defined in this set SHALL
     * be evaluated for each message submitted to this topic, and the
     * resultant total assessed fees SHALL be charged.<br/>
     * If this list is not empty, custom fees defined here SHALL be
     * charged _in addition to_ the base network and node fees.
     */
```
Note: I am guessing a bit as to behavior in this suggestion; please correct any errors above.